.github/workflows/oc-setup.yaml

```diff
@@ -0,0 +1,38 @@
+name: Install oc and set up the openshift context
```

Comment:
ok i think i get it - this makes the other yaml configs DRY since it seems to be used in the other files.